persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -9,9 +18,99 @@ CREATE TABLE IF NOT EXISTS users
     first_name      VARCHAR(36) NOT NULL,
     last_name       VARCHAR(36) NOT NULL,
     privilege_level VARCHAR(16) NOT NULL,
-    password_hash   BYTEA       NOT NULL
+    country_id      BIGINT,
+    password_hash   BYTEA       NOT NULL,
+
+    CONSTRAINT fk_user_country FOREIGN KEY (country_id) REFERENCES countries (id)
+);
+
+CREATE TABLE IF NOT EXISTS schools
+(
+    id         BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP,
+    name       VARCHAR(36) NOT NULL,
+    address    VARCHAR(64) NOT NULL,
+    hidden     BOOLEAN              DEFAULT FALSE,
+    country_id BIGINT,
+
+    CONSTRAINT fk_school_country FOREIGN KEY (country_id) REFERENCES countries (id)
+);
+
+CREATE TABLE IF NOT EXISTS school_additional_contacts
+(
+    id         BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP,
+    school_id  BIGINT      NOT NULL,
+    name       VARCHAR(36) NOT NULL,
+    email      VARCHAR(36),
+    address    VARCHAR(36),
+    phone      VARCHAR(36),
+
+    CONSTRAINT fk_school_contact_school FOREIGN KEY (school_id) REFERENCES schools (id)
 );
 
+CREATE TABLE IF NOT EXISTS school_reports
+(
+    id                                BIGSERIAL NOT NULL PRIMARY KEY,
+    created_at                        TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at                        TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at                        TIMESTAMP,
+    user_id                           BIGINT    NOT NULL,
+    school_id                         BIGINT    NOT NULL,
+    number_of_children                INT       NOT NULL,
+    contact_info_school               VARCHAR(128),
+    contact_info_principal            VARCHAR(128),
+    contact_info_literacy_coordinator VARCHAR(128),
+    number_of_books                   INT       NOT NULL,
+    most_recent_shipment_year         INT       NOT NULL,
+
+    CONSTRAINT fk_school_report_user FOREIGN KEY (user_id) REFERENCES users (id),
+    CONSTRAINT fk_school_report_school FOREIGN KEY (school_id) REFERENCES schools (id)
+);
+
+CREATE TABLE IF NOT EXISTS school_reports_with_libraries
```

Comment:
Do we still want to associate each report with some period of time, ie weekly/biweekly/etc? Regardless of when the data is submitted, I think it'll be valuable to associate some questions like "how many kids came to the library this week" with that week on a DB level.

Reply:
We could add a column to track the beginning of the time period the report is for? Does that sound reasonable?